src/annotator/sidebar.js

```diff
@@ -225,6 +218,18 @@ export default class Sidebar {
     this._emitter.destroy();
   }
 
+  /**
+   * Establish RPC channels from specific source frames
```

Comment:
```suggestion
   * Setup communication with a frame that has connected to the host.
```

I reworded this a little to try and describe logically what it does from the caller's point of view, rather than how it is implemented.

Reply:
Thanks